src/components/form/select/select.js

```diff
@@ -0,0 +1,161 @@
+import React from 'react';
+import styled, { css } from 'styled-components';
+
+const Select = ({ variant, title, optional, options, ...props }) => {
+    
+    const [selected, setSelected] = React.useState({ id : null, image : '/assets/icons/n_sample_img.svg', value : 'select' });
+    const [open, setOpen] = React.useState(false);
+
+    const toggleOpen = () => {
+        setOpen(!open);
+    }
+
+    const selectOption = (option) => {
+        setSelected(option);
+        toggleOpen();
+    }
+
+    return (
+        <Container>
+            <StyledLabel>{ title } <span hidden={ !optional }>(optional)</span> </StyledLabel>
+            <StyledSelect variant={ variant } { ...props } onClick={ toggleOpen }>
```

Comment:
```suggestion
            <StyledSelect variant={ variant } onClick={ toggleOpen }  { ...props }>
```

Always pass `props` in last in order to avoid them being overridden by the next props i.e `onClick` here